scripts/pre_commit_linter.py

```diff
@@ -340,6 +340,11 @@
     '/scripts/install_third_party.sh',
     '/.github/']
 
+NEW_CONVENTION_DIRECTORIES = [
```

Comment:
Instead of this, please use OLD_CONVENTION_DIRECTORIES and check "all frontend files minus stuff in OLD_CONVENTION_DIRECTORIES". Then, gradually reduce the list over time until we can remove it altogether.

Reply:
Done!